### PR TITLE
Added more commonly used basic commands.

### PR DESCRIPTION
**Pull request by ankurparihar**
Added pushd, popd, dirs, gzip, zip, unzip etc. commands

### CODE REVIEW

**Comment by ankurparihar** [comment]
Improvements.

File: README.md [hunk 1]
```diff
@@ -277,7 +286,8 @@ This are just examples. `chmod` has a lot of different configurations for differ
 ### Extracting compressed files
 
 `tar -xvzf <file.tar.gz>` - used to extract the .tar.gz file<br>
-
+`gzip -d <file_to_decompress>` - used to extract the .gz file<br>
```
Comment (CodeMaxx) [inline README.md]: Please add `gunzip <file_to_decompress>.gz` as well since it's very similar to this command

File: README.md [hunk 2]
```diff
@@ -290,7 +300,22 @@ This are just examples. `chmod` has a lot of different configurations for differ
 `-c`: Collects files to be compressed<br>
 `-v`: makes tar talk a lot. Verbose output shows you all the files being extracted.<br>
 `-z`: tells tar to decompress the archive using gzip<br>
-`-f`: this must be the last flag of the command, and the tar file must be immediately after. It tells tar the name and path of the compressed file.
+`-f`: this must be the last flag of the command, and the tar file must be immediately after. It tells tar the name and path of the compressed file.<br>
+
+`gzip file1 file2 file3` - used to compress any number of files into a .gz compressed format.<br>
```
Comment (CodeMaxx) [inline README.md]: Mention that this method compresses each of the files separately and not into a single .gz archive.

**Review by CodeMaxx — CHANGES_REQUESTED** [review]

**Comment by CodeMaxx** [comment]
I've added a couple of comments. The rests looks good. Thanks for using proper format for the Readme!

**Comment by ankurparihar** [comment]
Changes done.
Creating a new pull request.

**Pull request closed without merging**